compilation-config.yaml

```
@@ -21,6 +21,15 @@ fig_wg1_spm4:
   output_data_path: data/processed/fig-spm4/fig-spm4-timeseries.csv
   compile_function: openscm_ar6_wg1_data_compilation.compile_spm_fig_4_timeseries
 
+fig_wg1_spm7:
+  variables:
+      Cumulative Carbon Stored|Atmosphere:
```

```suggestion
      Cumulative Carbon Uptake|Atmosphere:
```

I would suggest using uptake as this is the term that is more commonly used (but not a massive deal either way)

---

Changed this to uptake